src/components/DeleteCardForm.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup";
+
+export default class DeleteCardForm extends Popup {
+  constructor(popupModalSelector, handleFormSubmit) {
+    super({ popupModalSelector });
+    this._popupModalSelector = document.querySelector(popupModalSelector);
```

Comment:
`this._popupModalSelector`

Word `selector` means a **string** that is used for searching elements in the DOM

Reply:
* You haven't corrected this remark https://github.com/divyaaa1812/se_project_aroundtheus/pull/1#discussion_r1271429507 

`this._popupModal`